router/src/main/resources/interceptor/README.md

```diff
@@ -39,3 +41,23 @@
     ```
 8. Copy interceptor certs to the APICTL project (i.e. Endpoint-certificates/interceptors).
 9. Start the docker-compose file.
+
+## Build Router Docker Image - for Integration/Local Testing
+
+Set `ROUTER_BASE_IMAGE` with the latest compatible Router docker image and `TARGET_ROUTER_IMAGE` with the image name that you want to be built.
+
+For example:
+- ROUTER_BASE_IMAGE=wso2/choreo-connect-router:1.1.0
+- TARGET_ROUTER_IMAGE=wso2/choreo-connect-router:1.2.0-m1-SNAPSHOT
+
+Change directory to here and execute the following script.
+
+```shell
+cat <<EOF > DockerfileAddInterceptor
+FROM $ROUTER_BASE_IMAGE
+COPY interceptor /home/wso2/interceptor
+EOF
+
+docker build -t $TARGET_ROUTER_IMAGE -f DockerfileAddInterceptor ../
```

Comment:
Is it `./`?
```suggestion
docker build -t $TARGET_ROUTER_IMAGE -f DockerfileAddInterceptor ./
```

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2978/commits/4b95f76810ac4716491351e69c9129084553f61d